examples/games/rock_paper_scissors/play_rock_paper_scissors.qnt

```diff
@@ -0,0 +1,57 @@
+// -*- mode: Bluespec; -*-
+module rock_paper_scissors {
+    // Constants for player names
+    const PLAYER1 : str
+    const PLAYER2 : str
+    // Enum types for the possible moves and game status
+    type Move = Init | Rock | Paper | scissors
+    type GameStatus = Started | Pending | Draw | Winner(str)
+    // Variables to hold the current state of each player's move and the game status
+    var p1State: Move
+    var p2State: Move
+    var status: GameStatus
+    def beats(m1: Move, m2: Move): bool = or {
```

Comment:
Thanks for removing the extra spaces! I'd add a few back tho, like one line before multi-line definitions such as this one
```suggestion

    def beats(m1: Move, m2: Move): bool = or {
```